sys-apps/shadow/shadow-4.10-r4.ebuild

```diff
@@ -234,10 +234,17 @@ pkg_preinst() {
 }
 
 pkg_postinst() {
+	# Missing entries from /etc/passwd can cause odd system blips.
+	# See bug #829872.
+	if ! pwck -r -q -R "${EROOT:-/}" &>/dev/null ; then
+		ewarn "Running 'pwck' returned errors. Please run it by hand and then"
+		ewarn "run 'pwconv' afterwards!"
```

Comment:
If the user corrects the shadow file using pwck, there should be no need to run pwconv.

Reply:
I think I got excited about being able to use the same text as below and didn't think. Yes, of course.